contracts/WildcardSteward_v3.sol

```diff
@@ -295,8 +295,9 @@ contract WildcardSteward_v3 is Initializable {
         uint256 _auctionStartPrice,
         uint256 _auctionEndPrice,
         uint256 _auctionLength
-    ) public notNullAddress(withdrawCheckerAdmin) {
+    ) public notNullAddress(_withdrawCheckerAdmin) {
         // This function effectively needs to call both _collectPatronage and _collectPatronagePatron from the v2 contract.
+        require(withdrawCheckerAdmin == address(0), "only can call once");
```

Comment:
This is also NB!!! Makes sure we can only call this function once and not multiple times...

Reply:
Yeah, I think there was confusion about checking the new add address is null and it not being set already! Good we caught this one